HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -398,9 +398,10 @@ def test_central_air_conditioners
     hpxml.cooling_systems[0].charge_defect_ratio = -0.11
     hpxml.cooling_systems[0].airflow_defect_ratio = -0.22
     hpxml.cooling_systems[0].cooling_capacity = 12345
+    hpxml.cooling_systems[0].cooling_efficiency_seer = 12.5
```

Comment:
For robustness, how about also including the year installed here? It will verify that the year installed has no impact on the test result when the efficiency is also provided. (Make the same change to other similar tests.)